presidio-anonymizer/presidio_anonymizer/operators/operators_factory.py

```diff
@@ -63,9 +63,18 @@ def get_deanonymizers() -> Dict[str, "Operator"]:
             )
         return OperatorsFactory._deanonymizers
 
+    @staticmethod
+    def __get_all_subclasses(cls):  # add this method
```

Comment:
```suggestion
    def __get_all_subclasses(cls: Type[Operator]):
```
I haven't tested this, please check, and an import `from typing import Type` is required.

Reply:
@omri374 Done.